nextflow_schema.json

```
@@ -65,6 +65,14 @@
           "default": 2000,
           "description": "Number of highly variable genes to use when performing dimensionality reduction on merged objects"
         },
+        "cell_type_blueprint_ref_file": {
+          "type": "string",
+          "default": "https://raw.githubusercontent.com/AlexsLemonade/OpenScPCA-analysis/e4ffb305a87a54133f398c4445ff92348fcb9d8a/analyses/cell-type-consensus/references/blueprint-mapped-ontologies.tsv",
```

To make this a bit easier to read and update, we can use the tag
```suggestion
          "default": "https://raw.githubusercontent.com/AlexsLemonade/OpenScPCA-analysis/refs/tags/v0.2.1/analyses/cell-type-consensus/references/blueprint-mapped-ontologies.tsv",
```